runtime/crab/src/lib.rs

```diff
@@ -144,14 +145,24 @@ where
 	}
 }
 
-impl_self_contained_call!();
-
 /// The version information used to identify this runtime when compiled natively.
 #[cfg(feature = "std")]
 pub fn native_version() -> NativeVersion {
 	NativeVersion { runtime_version: VERSION, can_author_with: Default::default() }
 }
 
+impl_self_contained_call!();
```

Comment:
BTW, I want to put it above the `sp_api::impl_runtime_apis! {`.

Reply:
I made some format updates as you suggested. Please check it again.